content/docs/portals.md

```diff
@@ -4,7 +4,7 @@ title: Portals
 permalink: docs/portals.html
 ---
 
-Portals provide a first-class way to render children into a DOM node that exists outside the DOM hierarchy of the parent component.
+Portals provide a first-class way to render children into a DOM node that exists outside the DOM hierarchy of the parent component.  They _must_ be returned from a component. The `createPortal` function is not a normal funciton that can be called from just anywhere.
```

Comment:
What does mean "just anywhere"?

Reply:
I'm stressing that this function will not work as expected if you call it from places other than as the `return` from a component.  One would think you could call this function from a helper, as a side-effect, etc. but it doesn't work when you call it from those other place (i.e. "just anywhere").  I'm submitting this addition (or similar if you think it can be said more clearly) because it's not obvious that this restriction exists.